### DISCUSSION TITLE
Remote app not working in single screen

### DISCUSSION THREAD

**Discussion by tao363** [FreeRDP/FreeRDP#10565] [Q&A] [answered]
The usemon:i: 0 parameter in the remoteapp doesn't work, I'm using remoteapp mode to start the remote app and can't control the app to only use one of my multiple monitors

**Comment by tao363** [comment]
> The usemon:i: 0 parameter in the remoteapp doesn't work, I'm using remoteapp mode to start the remote app and can't control the app to only use one of my multiple monitors

Why is it confusing to write parameters in the local rdp file, but not in settings in the freerdp project as a proxy?

**Reply by akallabeth** [reply]
remote app is what the name implies, restricting to a single monitor is kind of pointless.
you do not have a `monitor` but an application window and you can always move that around.

**Comment by tao363** [comment]
Thank you for your quick reply. You're right, but I want to directly control the resolution used by the remote app, but I'm limited to the resolution of the main screen when I choose a single screen, and the resolution of multiple screens can only be used in multi-screen cases. I want to make sure when did the configuration information of the rdp transfer get in freerdp? Especially the UseMultimon parameter.

**Reply by akallabeth** [reply, marked as answer]
your concept of `Remote APP` is flawed.
1. the application is running remotely 
2. the display of the application is locally

so, for this to work it requires the local setup to be mirrored so the application can work on all the screen.

if you want to know where this information is mirrored check the [\[MS-RDPEDISP\]](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpedisp/d2954508-f487-48bc-8731-39743e0854a9) and [\[MS-RDPBCGR\]](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/88186da8-2a0a-488e-ba8d-97af6cd89a3c) parts of the spec

it might also help if you tell us what you try to achieve as sometimes that is easier to understand than specific questions without appropriate context.

**Comment by tao363** [comment]
Thank you for your quick reply and helpful documents.
